### DISCUSSION TITLE
Github TLS certificate issue

### DISCUSSION THREAD

**Discussion by mjhajharia** [community/community#23577] [Other Feature Feedback, Questions, & Ideas]
<p>Hi! I have been trying to get github to issue a SSL certificate but it wont work, its been more than 24 hours, i even tried making a new repo and do the process, ive added and removed the CNAME a couple of times. all my host side records are correct and i do have two websites (<a href="http://notes.mjhajharia.com" rel="noopener nofollow ugc">notes.mjhajharia.com</a>) and mjhajharia[.]com working fine with github, but this one won’t somehow.</p>
<p>repo: <a href="https://github.com/mjhajharia/meenal-mjhajharia/settings/pages" rel="noopener nofollow ugc">https://github.com/mjhajharia/meenal-mjhajharia/settings/pages</a><br>
website → meenal[.]mjhajharia[.]com</p>
<p>(using the braces because i can only put 2 links in the post)</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="mjhajharia" data-post="1" data-topic="223633">
<div class="title">
<div class="quote-controls"></div>
 mjhajharia:</div>
<blockquote>
<p>repo: <a href="https://github.com/mjhajharia/meenal.mjhajharia.com/settings/pages">https://github.com/mjhajharia/meenal.mjhajharia.com/settings/pages</a></p>
</blockquote>
</aside>
<p>we cannot see that page because it’s not allowed, if you can tell us more, what is your domain registrar and other details,</p>
<p>this problem is so common here and always being resolved,</p>

**Comment by mjhajharia** [comment]
<p>my domain registrar is hostinger and the relevant website is <a href="http://meenal.mjhajharia.com" rel="noopener nofollow ugc">meenal.mjhajharia.com</a></p>
<p>here’s the CNAME record (the other 4 A records and CNAME for www are also present)<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/0/a00d7b5c7a73108d9fe828a6741b8014a7749183.png" data-download-href="/uploads/short-url/mPThmzCFPTLcjJI3cQRBu51aQ6v.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/55153708/181077998-0b562e5b-f2d9-4815-b73a-a74ab994e7d5.png" alt="image" data-base62-sha1="mPThmzCFPTLcjJI3cQRBu51aQ6v" width="690" height="58" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a00d7b5c7a73108d9fe828a6741b8014a7749183_2_690x58.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a00d7b5c7a73108d9fe828a6741b8014a7749183_2_1035x87.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a00d7b5c7a73108d9fe828a6741b8014a7749183_2_1380x116.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a00d7b5c7a73108d9fe828a6741b8014a7749183_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1684×142 4.79 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mjhajharia** [comment]
<p>from what i understand it’s a glitch from github because it’s stuck here (i already have to github pages sites with the same domain so this shouldnt be a problem afaik</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/9/696ee3860cdb4ab1462f177d1cd3dfa9c8e96d98.png" data-download-href="/uploads/short-url/f2HIaLDdVQcycNYaqYOyPdRXCyI.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/55153708/181077999-e7f26e2a-e4d7-4933-9d56-d3f6c93dabba.png" alt="image" data-base62-sha1="f2HIaLDdVQcycNYaqYOyPdRXCyI" width="690" height="309" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/9/696ee3860cdb4ab1462f177d1cd3dfa9c8e96d98_2_690x309.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/9/696ee3860cdb4ab1462f177d1cd3dfa9c8e96d98_2_1035x463.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/9/696ee3860cdb4ab1462f177d1cd3dfa9c8e96d98_2_1380x618.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/9/696ee3860cdb4ab1462f177d1cd3dfa9c8e96d98_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1662×746 89.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>how about the DNS Config?</p>

**Comment by mjhajharia** [comment]
<aside class="quote no-group" data-username="jdevstatic" data-post="5" data-topic="223633">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>DNS Config?</p>
</blockquote>
</aside>
<p>i’m not sure if this is what you mean<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/a/7af0c6518d58134f743d87fc42e2f1976ff5a548.png" data-download-href="/uploads/short-url/hxA6K7jYp4neMmadAXnyG9eZ2Na.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/55153708/181078001-76bad153-d9e6-4caf-ae1f-ed3cf1da4757.png" alt="image" data-base62-sha1="hxA6K7jYp4neMmadAXnyG9eZ2Na" width="690" height="40" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/a/7af0c6518d58134f743d87fc42e2f1976ff5a548_2_690x40.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/a/7af0c6518d58134f743d87fc42e2f1976ff5a548_2_1035x60.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/a/7af0c6518d58134f743d87fc42e2f1976ff5a548.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/a/7af0c6518d58134f743d87fc42e2f1976ff5a548_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1102×64 12.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><p></p>

**Comment by jfullstackdev** [comment]
<p>check this out, I’m not very sure of your subdomain, whether it’s being prevented to issue the certificate,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181078005-d93a4078-12ed-499c-b3ef-269fbf5225be.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages" target="_blank" rel="noopener">About custom domains and GitHub Pages - GitHub Docs</a></h3>

  <p>GitHub Pages supports using custom domains, or changing the root of your site's URL from the default, like octocat.github.io, to any domain you own.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by mjhajharia** [comment, marked as answer]
<p>i tried to add CAA records again to my website and verified the original domain <a href="http://mjhajharia.com" rel="noopener nofollow ugc">mjhajharia.com</a> once more and it got fixed, it was just a glitch i think, thanks anyway!</p>
